src/NUnitEngine/Addins/addin-tests/NUnitProjectLoaderTests.cs

```diff
@@ -86,10 +86,12 @@ public void LoadEmptyConfigs()
             }
         }
 
+        [TestCase("NUnitProject.nunit")]
+        [TestCase("NUnitProject_XmlDecl.nunit")]
         [Test]
```

Comment:
TestAttribute is now redundant. As a matter of style, we don't normally use it wiith test cases.